apps/src/templates/studioHomepages/TeacherHomepage.jsx

```diff
@@ -183,10 +183,7 @@ export default class TeacherHomepage extends Component {
         <ProtectedStatefulDiv ref="flashes" />
         <ProtectedStatefulDiv ref="teacherReminders" />
         {isEnglish && showSpecialAnnouncement && (
-          <SpecialAnnouncementActionBlock
-            hocLaunch={hocLaunch}
-            hasIncompleteApplication={!!sessionStorage['TeacherApplication']}
-          />
+          <SpecialAnnouncementActionBlock hocLaunch={hocLaunch} />
```

Comment:
For US-centric stuff like teacher applications restricting to English makes sense.  Do we want to remove the `isEnglish` check to show this HoC announcement in other languages too since it's relevant outside the US?

Reply:
I'm not sure - isn't CS Ed Week different in some other countries?  I'll ask marketing.